.github/workflows/update-charts.yml

```diff
@@ -63,31 +126,41 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - check-update-type
+      - setvariables
     if: needs.check-update-type.outputs.update_type == 'patch' && !endsWith(needs.check-update-type.outputs.repository, 'kwctl')
     permissions:
       contents: write
       pull-requests: write
     steps:
+      - name: create env vars from needs.setvariables.outputs
+        # This is neeeded because in scripts, we don't have a way to access the `needs` json object.
+        # The env vars defined here will be accesible in the scripts under `process.env.foo`
```

Comment:
It may be possible to drop this step, not save the vars as env vars, and in the script parse the `needs.setvariables.outputs` string into json.
So instead of  `let version = process.env.version` do
```let version = JSON.parse( ' ${{ needs.setvariables.outputs }} ' ).version```.
 But I couldn't get it to work.